fdbclient/NativeAPI.actor.cpp

```diff
@@ -1310,38 +1310,37 @@ ACTOR static Future<Void> backgroundGrvUpdater(DatabaseContext* cx) {
 	}
 }
 
+inline HealthMetrics populateHealthMetrics(const HealthMetrics& detailedMetrics, bool detailedOutput) {
```

Comment:
Should this function be marked `const`?

Reply:
I don't think we can do this. It's not a member function.